src/main/java/taxi/controller/driver/AddDriverController.java

```diff
@@ -25,8 +26,15 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp)
             throws IOException {
         String name = req.getParameter("name");
         String licenseNumber = req.getParameter("license_number");
-        Driver driver = new Driver(name, licenseNumber);
+        String login = req.getParameter("login");
+        String password = req.getParameter("password");
+        Driver driver = new Driver(name, licenseNumber, login, password);
         driverService.create(driver);
+        HttpSession session = req.getSession();
+        if (session.getAttribute("driver_id") == null) {
```

Comment:
I think, it would be better to  check in filter

Reply:
For example?
Just used session driver id for redirect logic after a new driver registration.